docs/tempo/website/traceql/construct-query.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Construct a TraceQL query
+menuTitle: Construct a query
+description: Learn how to construct a TraceQL query
+aliases:
+  - /docs/tempo/latest/traceql/construct-query
+weight: 100
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# Construct a TraceQL query
+
+In TraceQL, a query is an expression that is evaluated on one trace at a time. The query is structured as a set of chained expressions (a pipeline). Each expression in the pipeline selects or discards spansets from being included in the results set. For example:
+
+```
+{ .http.method = "POST" } | by(.http.target) | count() > 2
```

Comment:
`by()` doesn't work yet. would prefer focusing on what we can do. everything else works.